views/default/elements/navigation.css.php

```diff
@@ -5,6 +5,43 @@
  */
 
 /* ***************************************
+	ANCHORS & BADGES
+*************************************** */
+
+.elgg-anchor * {
+	display: inline;
+	vertical-align: middle;
+}
+
+.elgg-anchor * + .elgg-anchor-label {
+	margin-left: 5px;
+}
+
+.elgg-badge {
+	display: inline-block;
+	margin-left: 5px;
+	color: #fff;
+	background-color: #777;
+	padding: 3px 7px;
+	line-height: 1;
+	text-align: center;
+	white-space: nowrap;
+	vertical-align: middle;
+	border-radius: 10px;
+	font-size: 90%;
+}
+
+.elgg-menu-topbar .elgg-badge {
```

Comment:
I thought about adding a `badge_type` option that would produce `.elgg-badge-$badge_type` classes, but it seems like you'll always need some contextual styles anyway.